packages/fxa-content-server/server/lib/sentry.js

```diff
@@ -91,6 +91,17 @@ function tryCaptureValidationError(err) {
     }
 
     if (errorDetails) {
+      // We are ignoring all navigation timings issues until we come up with a better
+      // solution for capturing timing data. See FXA-7005.
+      if (
+        typeof errorDetails.details.some === 'function' &&
+        errorDetails.details.some(
+          (detail) => (detail.message || '').indexOf('navigationTiming.') >= 0
```

Comment:
You could also use [`String#includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) [which returns a Boolean] instead of `.indexOf()`:
```js
(detail) => (detail.message || '').includes('navigationTiming.')
```